docs/modules/operation/pages/quick-start/visualize-data.adoc

```diff
@@ -0,0 +1,99 @@
+. Click *Add Dashlet* and configure its settings:
+** *Title:* dashlet name to display in the ops board.
+** *Priority:* how often the dashlet displays in the rotation.
+Priority 1 is the highest, meaning it appears the most often.
+** *Duration:* how long the dashlet displays in the rotation, in seconds.
+** *Boost-Priority:* change the dashlet's priority if it is in the alert state.
+This setting is optional, and is not available for all dashlets.
+** *Boost-Duration:* change the display duration if the dashlet is in the alert state.
+This setting is optional, and is not available for all dashlets.
+** (Optional) Click *Properties* to configure additional settings (alarm severity, chart type, and so on).
+. Click the *Up Arrow* and *Down Arrow* symbols to change the dashlet's order.
+This affects its rotation order in the Ops Board view, or its position in the Ops Panel view.
+. Click *Preview* in the dashlet settings area to preview the dashlet.
+. Click *Preview* beside the operator board name to preview the board and all of its dashlets.
+
+The board's configuration is automatically saved.
+To view the ops board, click menu:Dashboards[Ops Board] in the menu bar.
+
+For more information about ops boards and dashlets, see xref:operation:deep-dive/admin/webui/opsboard/introduction.adoc[Operator Board] in the Deep Dive section.
+
+== Surveillance View
+
+The Surveillance View is similar to its component of the same name in the <<#qs-visualize-dashboard, Dashboard>>.
+You can view it by clicking menu:Status[Surveillance] in the top menu bar.
+
+Just like in the Dashboard, you can select rows, columns, and cells to narrow your data focus.
+
+For more information, see xref:operation:deep-dive/admin/webui/surveillance-view.adoc[Surveillance View] in the Deep Dive section.
+
+== Heatmap
+
+You can use the heatmap to display either unacknowledged alarms or ongoing node outages.
+Each visualization can be applied to categories, foreign sources, or node services.
+
+You can view the heatmap by clicking menu:Status[Heatmap] in the top menu bar.
+
+For more information, see xref:operation:deep-dive/admin/webui/heatmap.adoc[Heatmap] in the Deep Dive section.
+
+== Trend
+
+The Trend page displays small, inline charts of database-related statistics (nodes experiencing outages, unacknowledged alarms, and so on).
+You can view it by clicking menu:Status[Trend] in the top menu bar.
+
+For more information, see xref:operation:deep-dive/admin/webui/trends.adoc[Trend] in the Deep Dive section.
+
+== Beyond Quick Start
+
+Refer to xref:operation:deep-dive/admin/webui/introduction.adoc[Administrative Web Interface] in the Deep Dive section for more information on visualizations, including the following:
+
+* xref:operation:deep-dive/admin/webui/opsboard/criteria-builder.adoc[Filtering dashlet information using the Criteria Builder].
+* xref:operation:deep-dive/admin/webui/ui-preview.adoc[Using the UI preview].
```

Comment:
Let's keep this for now, but we may need to remove it as part of the H31 work to get rid of the UI preview and incorporate those screens into the product.